src/hotspot/share/opto/escape.cpp

```diff
@@ -883,6 +872,10 @@ bool ConnectionGraph::add_final_edges_unsafe_access(Node* n, uint opcode) {
     }
     return true;
   }
+#ifdef ASSERT
+    n->dump(1);
+    assert(false, "not unsafe");
```

Comment:
Indentation is wrong.

Reply:
Thanks for the review, Tobias! All changed.